doc/src/manual/noteworthy-differences.md

```diff
@@ -237,6 +237,16 @@ For users coming to Julia from R, these are some noteworthy differences:
   * In Julia, keyword arguments must be passed using keywords, unlike Python in which it is usually possible
     to pass them positionally. Attempting to pass a keyword argument positionally alters the method
     signature leading to a `MethodError` or calling of the wrong method.
+  * Conversely, positional arguments cannot be given by name in Julia.
+    If a function's signature is e.g. `lognormal(median, g)`,
+    then calling it as `lognormal(median = 4, g = 1)` will not work.
+    Instead, you can explain arguments by name on separate lines:
+    ```
+    median = 4
+    g      = 1  # Geometric standard deviation
+    lognormal(median, g)
+    ```
```

Comment:
A slightly more compact option might be e.g.
`However, you are free to encode this information in variable names:`
```
median, g = 4, 1
lognormal(median, g)
```
I think without the inline comments example is fine too